frontend/public/locales/en/personal-information.json

```diff
@@ -24,7 +24,7 @@
       "breadcrumbs": {
         "home": "Home",
         "personal-information": "Personal information",
-        "home-address-change": "Change home address"
+        "suggested-address": "Suggested address"
```

Comment:
This change should be reverted as we're modifying the breadcrumb on the "Change home address" page

Reply:
done